tests/functional/tests/management/test_composite_volume.py

```diff
@@ -363,4 +385,58 @@ def test_load(pyocf_ctx):
     requirements:
       - composite_volume::cache_attach_load
     """
-    pass
+    vols = [RamVolume(S.from_MiB(3)) for _ in range(16)]
+    cvol = CVolume(pyocf_ctx)
+    for vol in vols:
+        cvol.add(vol)
+
+    cache = Cache.start_on_device(cvol)
+
+    core_device = RamVolume(S.from_MiB(20))
+    core = Core.using_device(core_device)
+
+    cache.add_core(core)
+
+    vol = CoreVolume(core, open=True)
+    write_data = Data.from_string("Verify load")
+    io = vol.new_io(
+        cache.get_default_queue(), S.from_sector(3).B, write_data.size, IoDir.WRITE, 0, 0
+    )
+    io.set_data(write_data)
+
+    cmpl = OcfCompletion([("err", c_int)])
+    io.callback = cmpl.callback
+    io.submit()
+    cmpl.wait()
+    assert 0 == int(cmpl.results["err"])
+
+    cache.stop()
+
+    cvol = CVolume(pyocf_ctx)
+    for v in vols:
+        cvol.add(v)
```

Comment:
Is this necessary to create this composite volume again? Can't the test just use `cvol` again?

Reply:
Yes it was an issue. Without second initialization segfault occurs.
During cache.stop() there is some deinitialization of cvolume.